src/towncrier/test/helpers.py

```diff
@@ -69,3 +70,43 @@ def setup_simple_project(
 
     if mkdir_newsfragments:
         Path("foo/newsfragments").mkdir()
+
+
+def with_isolated_simple_project(
+    *,
+    with_git=False,
+    config: str | None = None,
+    extra_config: str = "",
```

Comment:
Just a comment. Not sure if it make sense to keep this.

I find it easier to read a test, if it has the full configuration.